strongbox-client/src/test/java/org/carlspring/strongbox/client/ArtifactResolverIntegrationTest.java

```diff
@@ -86,9 +85,16 @@ public MultiHttpClientConnThread(RestArtifactResolver artifactResolver,
         @Override
         public final void run()
         {
-            CloseableRestResponse response = artifactResolver.get(url);
-            ResourceCloser.close(response, LOGGER);
+            try (CloseableRestResponse response = artifactResolver.get(url))
+            {
+                // Left empty because the original method had no other logic than the now-removed
+                // closure of a CloseableRestResponse
+            }
+            catch (Exception e)
+            {
+                LOGGER.error(e.getMessage(), e);
```

Comment:
Cosmetic: Our loggers across the codebase are lowercase. Could you please change this? Thanks! :)

Reply:
Hi there @carlspring , 
Thanks for all the comments, I will change the code